### ISSUE TITLE
Bug (Lizzie v. 5): layout too far left (just a little annoyance)

### ISSUE THREAD

**Issue by WYVYW** [featurecat/lizzie#264]
- layout too far left (just a little annoyance):
 winrate graph is only partially visible (fullscreen or windowed on 1366x768)
 subboard is the same problem but only when using fullscreen (stones on a line only half visible)

**Comment by featurecat** [comment]
yep this is an inset issue: window frames often overlap a small portion of the x coordinate with a border, which covers up the contents of the window. we can fix this when we refactor everything, so that drawing objects on the screen accounts for the window border.

**Comment by featurecat** [comment]
on my computer (ubuntu) this is not a problem, since my windows don't have any borders, so I didn't fix this earlier :)

**Comment by featurecat** [comment]
no please leave it open, we haven't fixed this yet

**Comment by WYVYW** [comment]
Ok, i thought it was not seen as a bug, since it did not get a bug label.

**Comment by featurecat** [comment]
oh.. sorry, I haven't given a lot of things labels recently. I'm so busy. The mobile github client doesn't make labeling things easy.

**Comment by OlivierBlanvillain** [comment]
Is this fixed? @WYVYW do you have a screenshot to share?

**Comment by WYVYW** [comment]
It seems partly fixed. One might see it here only in the bottom left. The subboard is only partly visible.
![image](https://user-images.githubusercontent.com/38362247/46389821-a184c700-c6d4-11e8-9eba-e2b79da95d1d.png)
